### PR TITLE
Add Footer component

### PR DESCRIPTION
**Pull request by sindhujaponnuru**
Closes #16

### CODE REVIEW

File: src/components/footer/footer.js [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+    icons: contactIcons,
+  },
+  {
+    title: 'Follow us',
+    icons: socialIcons,
+  },
+]
+
+const Footer = () => {
+  const styles = useStyles()
```
Comment (pinnamanivenkat) [inline src/components/footer/footer.js]: ; required

File: src/components/footer/footer.js [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+  return (
+    <footer className={styles.footer}>
+      <Paper className={styles.paper}>
+        <Grid container spacing={2} justify="center" alignItems="center">
+          {footerItems.map(item => (
+            <FooterItem name={item.title} icons={item.icons}></FooterItem>
```
Comment (pinnamanivenkat) [inline src/components/footer/footer.js]: key required

File: package.json [hunk 3]
```diff
@@ -5,6 +5,8 @@
   "version": "0.1.0",
   "author": "turing hut",
   "dependencies": {
+    "@material-ui/core": "^4.11.0",
+    "@material-ui/icons": "^4.9.1",
     "gatsby": "2.23.12",
```
Comment (abhishekthalatoty) [inline package.json:11]: Remove "^"

File: src/components/footer/footer-item.js [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+        <Grid item xs>
+          <a href={route} alt={title}>
+            {icon}
+          </a>
+        </Grid>
+      ))}
+    </Grid>
+  )
+}
+export default FooterItem
```
Comment (abhishekthalatoty) [inline src/components/footer/footer-item.js:37]: Destructuring the Props would be a Better way.

File: src/pages/index.js [hunk 5]
```diff
@@ -1,5 +1,10 @@
 import React from 'react'
+import Footer from '../components/footer/footer'
 
-const IndexPage = () => <h1>Hello world!</h1>
+const IndexPage = () => (
+  <main>
+    <Footer></Footer>
```
Comment (abhishekthalatoty) [inline src/pages/index.js]: You can replace Footer with a Self Closing Tag

File: src/components/footer/footer.js [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+
+const Footer = () => {
+  const styles = footerStyles()
+  return (
+    <footer className={styles.footer}>
+      <Paper className={styles.paper}>
+        <Grid container spacing={2} justify="center" alignItems="center">
+          {footerItems.map(item => (
+            <FooterItem name={item.title} icons={item.icons}></FooterItem>
+          ))}
```
Comment (abhishekthalatoty) [inline src/components/footer/footer.js:62]: FooterItem Tag can be replaced with a Self Closing Tag

File: src/pages/index.js (hunk 5 above)
Comment (sindhujaponnuru) [reply src/pages/index.js]: okay sure

File: src/components/footer/footer.js (hunk 6 above)
Comment (sindhujaponnuru) [reply src/components/footer/footer.js:62]: okay!

File: package.json (hunk 3 above)
Comment (sindhujaponnuru) [reply package.json:11]: okay!

File: src/components/footer/footer-item.js (hunk 4 above)
Comment (sindhujaponnuru) [reply src/components/footer/footer-item.js:37]: instead of exporting the component?

File: src/components/footer/footer-item.js [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+    </Grid>
+  )
+}
+
+const IconList = props => {
+  return (
+    <Grid container spacing={1}>
+      {props.icons.map(({ title, icon, route }) => (
+        <Grid item xs>
+          <a href={route} alt={title}>
```
Comment (pinnamanivenkat) [inline src/components/footer/footer-item.js:22]: Perhaps add a circle background.

File: src/components/footer/footer.js [hunk 8]
```diff
@@ -0,0 +1,86 @@
...
+      },
+    ],
+  },
+  {
+    title: 'Contact us',
+    icons: [
+      {
+        title: 'Email',
+        icon: <Email />,
+        route: 'www.google.com',
```
Comment (pinnamanivenkat) [inline src/components/footer/footer.js]: Please remove these placeholders and add @mailto

File: src/components/footer/footer.js [hunk 9]
```diff
@@ -0,0 +1,86 @@
...
+      },
+      {
+        title: 'Phone',
+        icon: <LocalPhone />,
+        route: 'www.google.com',
```
Comment (pinnamanivenkat) [inline src/components/footer/footer.js]: similar for the phone.

File: src/components/footer/footer.js [hunk 10]
```diff
@@ -0,0 +1,86 @@
...
+import FooterItem from './footer-item'
+
+const footerItems = [
+  {
+    title: 'Find us on Maps',
+    icons: [
+      {
+        title: 'Maps',
+        icon: <Map />,
+        route: 'www.google.com',
```
Comment (pinnamanivenkat) [inline src/components/footer/footer.js]: add all the working, proper routes.

**Review by pinnamanivenkat — CHANGES_REQUESTED** [review]

File: src/components/footer/footer.js (hunk 10 above)
Comment (sindhujaponnuru) [reply src/components/footer/footer.js]: Done

**Comment by sindhujaponnuru** [comment]
Screenshot (16)

**Review by pinnamanivenkat — APPROVED** [review]

**Pull request merged**